modules/programs/most.nix

```diff
@@ -0,0 +1,22 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.most;
+
+in {
+  options.programs.most = {
+    enable = mkEnableOption "Most, a powerful paging program";
+    sessionPager =
+      mkEnableOption "use Most as the session pager, e.g. for manpages";
```

Comment:
I think this option could be removed. Presumably, if you enable the module you would always want Most as pager?